_posts/ja/newsletters/2019-10-16-newsletter-ja.md

```diff
@@ -20,7 +20,7 @@ lang: ja
 - **Eclair 0.3.2へのアップグレード:** [Eclair's newest release][eclair 0.3.2]
  は、バックアップが改善され、ゴシップデータの同期がより帯域幅効率的になります（特にモバイルデバイス上のノードなどの非ルーティングノード用）。またその他の多くの新機能とバグ修正が含まれています。
 
-- **リリース候補（RC）のレビュー:** 2つの主要なBitcoinインフラストラクチャプロジェクトが、ソフトウェアの次のバージョンのリリース候補（RC）を発表しました。一般公開前に問題を発見して修正できるよう、開発者と経験豊富なユーザーはこれらのRCのテストを支援するようお願いします。
+- **<!--n1-->リリース候補（RC）のレビュー:** 2つの主要なBitcoinインフラストラクチャプロジェクトが、ソフトウェアの次のバージョンのリリース候補（RC）を発表しました。一般公開前に問題を発見して修正できるよう、開発者と経験豊富なユーザーはこれらのRCのテストを支援するようお願いします。
```

Comment:
I'm not sure this is needed, since there are no other anchors on this page called with id `#rc`. The link currently works on the live site: https://bitcoinops.org/ja/newsletters/2019/10/16/#rc

How did you find where to add these comments?

Reply:
> How did you find where to add these comments?

The failing HTMLProofer test identified the broken file but not the line number; I opened the files and guess where the links were broken.  Looks like I guessed wrong here.  I'll back out this particular change.